node/test/Test/Pos/Client/Txp/UtilSpec.hs

```diff
@@ -275,6 +274,7 @@ makeManyUtxoTo1Params numFrom amountEachFrom amountTo = CreateMTxParams {..}
             k <- [0..numFrom-1]]
     cmpSigners = one (fakeSigner sk, addr)
     cmpOutputs = one txOutAuxOutput
+    cmpInputSelectionPolicy = OptimizeForHighThroughput
```

Comment:
This is a change. Did we want to change it? Before we used `cmpInputSelectionPolicy` we used `useGroupedInputs = OptimizeForSecurity` everwhere.